test/Microsoft.Extensions.Internal.Test/CopyOnWriteDictionaryHolderTest.cs

```diff
@@ -39,6 +42,9 @@ public void ReadOperation_DelegatesToSourceDictionary_IfNoMutationsArePerformed(
             Assert.False(holder.HasBeenCopied);
             Assert.Same(source, holder.ReadDictionary);
         }
+#elif NETCOREAPP2_0
+#else
+#error Target framework needs to be updated
```

Comment:
It means we'll have compile errors if we miss updating these as part of future TFM changes.